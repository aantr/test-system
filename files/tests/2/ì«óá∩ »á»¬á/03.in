28/02/2000