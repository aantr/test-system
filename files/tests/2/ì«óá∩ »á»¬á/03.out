29/02/2012